package.js

```diff
@@ -3,19 +3,19 @@
 Package.describe({
   name: 'mizzao:user-status',
   summary: 'User connection and idle state tracking for Meteor',
-  version: '1.0.1',
-  git: 'https://github.com/mizzao/meteor-user-status.git'
+  version: '1.0.2',
+  git: 'https://github.com/Meteor-Community-Packages/meteor-user-status.git'
 });
 
 Package.onUse((api) => {
-  api.versionsFrom(['1.7.0.5', '2.3']);
+  api.versionsFrom(['1.7.0.5', '2.7']);
```

Comment:
Might need to look at this versionsFrom.

Reply:
We can leave it for this patch release or bump to `1.12.1` as anything before that is just too old.